app/model/cvegroup.py

```diff
@@ -4,7 +4,7 @@
 
 
 pkgname_regex = r'^([a-z\d@\.\_\+-]+)$'
-pkgver_regex = r'^(\d+:)?([\w]+[\._+]*)+\-\d+$'
+pkgver_regex = r'^(\d+:)?([\w]+[\._+]*)+\-\d[\d\.]*$'
```

Comment:
If I'm not mistaken, this regex would allow trailing dots as well. Perhaps something like `r'^(\d+:)?([\w]+[\._+]*)+\-\d+(\.\d+)*$'` would be better?